docs/Failsafe.md

```diff
@@ -103,7 +103,7 @@ Configure the RC switched failsafe action. It can be one of:
 
 Time throttle level must have been below 'min_throttle' to _only disarm_ instead of _full failsafe procedure_.
 
-Use standard RX usec values.  See [Rx documentation](Rx.md).
+Use standard RX used values.  See [Rx documentation](Rx.md).
```

Comment:
Actually I think this is supposed to be "usec" as in microseconds.

Reply:
Interesting ! But then it should be `microsecond values` here

Reply:
Good catch! I suggest I put `us`, which is already used in cli.md. The other option would be to use the standard SI prefix, `μs`.

Reply:
Prefer the latter. @klutvott123 ?